monai/transforms/spatial/dictionary.py

```diff
@@ -832,7 +830,7 @@ def __call__(
 
         if self._do_transform:
             grid = self.rand_2d_elastic.deform_grid(spatial_size=sp_size)
-            grid = self.rand_2d_elastic.rand_affine_grid(grid=grid)
+            grid, _ = self.rand_2d_elastic.rand_affine_grid(grid=grid)
```

Comment:
Hi @rijobro , you can replace the "_" with "affine" for your inverse logic in this transform later.